airflow/models/taskinstance.py

```diff
@@ -306,6 +306,7 @@ def _run_raw_task(
             ti.state = TaskInstanceState.SKIPPED
             _run_finished_callback(callbacks=ti.task.on_skipped_callback, context=context)
             TaskInstance.save_to_db(ti=ti, session=session)
+            return None
```

Comment:
Here I finished the flow successfully but wondering if better raising here the exception.